tests/experimental/datasources/test_postgres_datasource.py

```diff
@@ -132,15 +139,10 @@ def test_construct_table_asset_directly_with_no_splitter(create_source):
 @pytest.mark.unit
 def test_construct_table_asset_directly_with_splitter(create_source):
     with create_source(lambda _: None) as source:
-        splitter = ColumnSplitter(
-            method_name="splitter_method",
-            column_name="col",
-            param_defaults={"a": [1, 2, 3], "b": list(range(1, 13))},
-        )
         asset = TableAsset(
             name="my_asset",
             table_name="my_table",
-            column_splitter=splitter,
+            column_splitter=SqlYearMonthSplitter(column_name="col"),
```

Comment:
Here I use the new `SqlYearMonthSplitter` since `ColumnSplitter` is now abstract.